common/collect_ovf_vm_config.yml

```diff
@@ -247,3 +249,13 @@
   ansible.builtin.copy:
     dest: "{{ ovf_vm_hardware_config_path }}/{{ ovf_vm_hardware_config_file }}"
     content: "{{ [ovf_vm_hardware_config] | to_nice_json }}"
+
+- name: "Display ddb.toolsVersion of open-vm-tools in OVA"
+  debug: var=ovf_vm_vmtools_info['Config Tools Version']
```

Comment:
You can remove this debug message but add a "success_msg" to below assert task, like:
```
success_msg: "The The ddb.toolsVersion in OVA template is {{ ovf_vm_vmtools_info['Config Tools Version'] }}, which is as expected."
```

Reply:
Change it. Thanks